src/test/java/com/alipay/remoting/demo/BasicUsageDemoByJunitWithIpv6.java

```diff
@@ -0,0 +1,287 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.remoting.demo;
+
+import com.alipay.remoting.Connection;
+import com.alipay.remoting.ConnectionEventType;
+import com.alipay.remoting.InvokeCallback;
+import com.alipay.remoting.exception.RemotingException;
+import com.alipay.remoting.rpc.RpcClient;
+import com.alipay.remoting.rpc.RpcResponseFuture;
+import com.alipay.remoting.rpc.common.BoltServer;
+import com.alipay.remoting.rpc.common.CONNECTEventProcessor;
+import com.alipay.remoting.rpc.common.DISCONNECTEventProcessor;
+import com.alipay.remoting.rpc.common.PortScan;
+import com.alipay.remoting.rpc.common.RequestBody;
+import com.alipay.remoting.rpc.common.SimpleClientUserProcessor;
+import com.alipay.remoting.rpc.common.SimpleServerUserProcessor;
+import com.alipay.remoting.util.RemotingUtil;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+/**
+ * basic usage demo
+ * 
+ * basic usage of rpc client and rpc server with ipv6
+ * 
+ * @author xiaomin.cxm
+ * @version $Id: BasicUsageDemo.java, v 0.1 Apr 6, 2016 8:58:36 PM xiaomin.cxm Exp $
+ */
+public class BasicUsageDemoByJunitWithIpv6 {
```

Comment:
Is this a demo program or a test program? If it's a demo program, I suppose we won't have methods annotated with `@Test`? If it's a test program, then I think the file name should be like `XXXTest.java`?

Reply:
this file is modified from "BasicUsageDemoByJunit", a demo by Junit, so keep the same style.
it has been deleted for it is a unnecessary UT.